build.sh

```diff
@@ -3,7 +3,7 @@ set -ex
 
 ESBUILD="node_modules/.bin/esbuild --bundle"
 
-$ESBUILD --format=iife --target=chrome90 --minify playwright/content.ts --outfile=dist/autoconsent.playwright.js
+$ESBUILD --format=iife --target=es2021 --minify playwright/content.ts --outfile=dist/autoconsent.playwright.js
```

Comment:
Why do we need to change this? Are we missing some syntax support that's needed? It looks to be a safe change, as this is just for playwright, but perhaps we can split it into a separate PR?

Reply:
This is needed because otherwise optional chaining is transpiled weirdly and the new eval snippets don't work in the tests